usr/share/rear/lib/rear-shell.bashrc

```diff
@@ -20,7 +22,7 @@ unset REAR_EVAL # reduce environmental pollution
 VERBOSE=1
 
 # source Relax-and-Recover functions
-for script in $SHARE_DIR/lib/*functions.sh ; do source $script ; done
+for script in $SHARE_DIR/lib/*functions.sh; do source $script; done
```

Comment:
please edit the commit to remove this unrelated whitespace change (which is contrary to examples in https://github.com/rear/rear/wiki/Coding-Style as well )